### DISCUSSION TITLE
Make error on missing task definition ignorable

### DISCUSSION THREAD

**Discussion by Alfred-Mountfield** [vercel/turborepo#7472] [Ideas]
### Which project is this feature idea for?

Turborepo

### Describe the feature you'd like to request

When you call `turbo <TASK>`, if turbo can't find the task definition it returns an error:
```
$ turbo my-task
 ERROR  run failed: error preparing engine: Could not find the following tasks in project: my-task
Turbo error: error preparing engine: Could not find the following tasks in project: my-task
```

This makes sense as default behaviour, but can be frustrating if you want to use it within a monorepo to call tasks on `postinstall` (for example) but then also wish to utilise `turbo prune`. 

How I wish to use this:
- I have some packages that include Python
- I wish to have `package.json` scripts within them to setup and install their Python dependencies in their venvs
- I wish to call those scripts in `postinstall` in the root
- When building a docker image of an unrelated package, I'd like to call `turbo prune other-package`
- When trying to build that package, it now fails on `postinstall` with the above message.

### Describe the solution you'd like

An argument you can provide to turbo to ignore missing task definitions. Perhaps `--ignore-missing` or something like that. Or perhaps better than that, an environment variable so you can set it in certain circumstances (such as after a `turbo prune`) without it being in your `package.json` all the time.

### Describe alternatives you've considered

I'm unsure what way to get around this other than somehow grepping the output which doesn't seem ideal, or creating a no-op equivalent script in every package.

**Comment by mehulkar** [comment]
Can you provide a small reproduction I can play around with? The `postinstall` usage is an interesting use case, but I'm not exactly sure how it's setup. If the packages aren't in the `prune` output, I would expect that `turbo run` would be able to ignore them.

**Comment by Megamannen** [comment]
A workaround is to add a `no-op` postinstall in the ones that doesn't need one 


```json 
"postinstall": "echo 'No postinstall'"
```
